### PR TITLE
Loanexplorer fields lw

### PR DESCRIPTION
**Pull request by ros3bud**
There are lots of standard field updates here for the Loan Explorer integration. Here is a link to the list I used: https://docs.google.com/spreadsheets/d/1apSQQNpNWHMdcqEoFyQv3jXGswieg-ibGX3B-ju0pXw/edit#gid=136066529. 

LGTY?

### CODE REVIEW

File: fields/mortgage.json [hunk 1]
```diff
@@ -1,48 +1,63 @@
...
 { "id": "mortgage.application.monthly_debt_payment",    "name": "Mortgage Application Monthly Debt Payment",  "type": "number",      "description": "Amount paid monthly against outstanding debts", "examples": ["1000", "$200", "3,000", "40"] }
 { "id": "mortgage.application.late_payments",           "name": "Mortgage Application Number Late Payments",   "type": "number",      "description": "How many late mortgage payments have been made?", "examples": ["1", "2", "3", "4"] }
 { "id": "mortgage.bankruptcy.was_declared",             "name": "Mortgage Application Bankruptcy Was Declared","type": "boolean",     "description": "Bankruptcy was declared at some point in the past" }
-{ "id": "mortgage.bankruptcy.discharged_months_ago",    "name": "Mortgage Application Bankruptcy Discharged Months Ago","type": "range", "description": "Number of months ago that bankruptcy was discharged" }
+{ "id": "mortgage.bankruptcy.discharged_months_ago",    "name": "Mortgage Application Bankruptcy Discharged Months Ago","type": "range", "description": "Number of months since last bankruptcy discharged occured" }
```
Comment (cgrayson) [inline fields/mortgage.json]: Typos in description.

File: fields/mortgage.json [hunk 2]
```diff
@@ -1,48 +1,63 @@
...
+{ "id": "mortgage.first_mortgage_year",                 "name": "Mortgage First Mortgage Year",                "type": "number",     "description": "The origination year of the loan on the property" }
 { "id": "mortgage.foreclosure.was_filed",               "name": "Mortgage Application Foreclosure Was Filed",  "type": "boolean",     "description": "Foreclosure was filed at some point in the past" }
-{ "id": "mortgage.foreclosure.filed_months_ago",    "name": "Mortgage Application Foreclosure Filed Months Ago","type": "range", "description": "Number of months ago that foreclosure was filed" }
+{ "id": "mortgage.foreclosure.filed_months_ago",        "name": "Mortgage Application Foreclosure Filed Months Ago","type": "range", "description": "Number of months ago that foreclosure was filed" }
+{ "id": "mortgage.foreclosure.discharged_months_ago",   "name": "Mortgage Application Foreclosure Discharged Months Ago","type": "range", "description": "Number of months since the last discharge occurred.", "examples": ["1-12", "25-36", "40", "50 to 60"] }
```
Comment (cgrayson) [inline fields/mortgage.json]: I noticed there are `examples` here, but not on other "months_ago" fields.

Comment (ros3bud) [inline fields/mortgage.json]: Yeah, I went back and forth about whether examples should be there. What do you think?

File: fields/mortgage.json [hunk 3]
```diff
@@ -1,48 +1,63 @@
...
-{ "id": "mortgage.loan.is_jumbo",                       "name": "Mortgage Loan Is Jumbo",                      "type": "boolean",     "description": "Whether the loan is a non-conforming jumbo loan"}
-{ "id": "mortgage.loan.lifetime_cap",                   "name": "Mortgage Loan Lifetime Cap",                  "type": "number",      "description": "The lifetime adjustment cap"}
-{ "id": "mortgage.loan.lock_in_days",                   "name": "Mortgage Loan Lock In Days",                  "type": "number",      "description": "Then number of days that the lender assumes the rate will be locked for"}
+{ "id": "mortgage.loan.due_in_months",                  "name": "Mortgage Loan Due In Months",                 "type": "number",      "description": "The number of months until the loan is due" }
+{ "id": "mortgage.loan.fees",                           "name": "Mortgage Loan Fees",                          "type": "array",       "description": "The fees associated with the loan" }
```
Comment (cgrayson) [inline fields/mortgage.json]: I see this is of type `array`, maybe specify in the description what it's an array of (e.g., numbers, strings), or give examples?

File: fields/mortgage.json [hunk 4]
```diff
@@ -1,48 +1,63 @@
...
+{ "id": "mortgage.loan.fee_included_in_apr",            "name": "Mortgage Loan Fee Included In APR",           "type": "boolean",     "description": "Is the fee is included in the APR?" }
```
Comment (cgrayson) [inline fields/mortgage.json]: I think it's worthwhile, if possible, to stick to the convention of using `is_` in boolean field ids.

File: fields/mortgage.json [hunk 5]
```diff
@@ -1,48 +1,63 @@
...
 { "id": "mortgage.loan.purpose",                        "name": "Mortgage Loan Purpose",                       "type": "string",      "description": "Purpose of loan","examples": ["refi", "purchase", "equity"] }
-{ "id": "mortgage.loan.rate_adjustment_period_in_months","name":"Mortgage Loan Rate Adjustment Period",        "type": "number",      "description": "The interval between rate adjustments, in months"}
-{ "id": "mortgage.loan.term",                           "name": "Mortgage Loan Term",                          "type": "number",      "description": "Number of years in term","examples":["15","30"] }
-{ "id": "mortgage.loan.term_in_months",                 "name": "Mortgage Loan Term",                          "type": "number",      "description": ""}
+{ "id": "mortgage.loan.rate_adjustment_period_in_months","name":"Mortgage Loan Rate Adjustment Period",        "type": "number",      "description": "The interval between rate adjustments, in months" }
+{ "id": "mortgage.loan.term",                           "name": "Mortgage Loan Term",                          "type": "number",      "description": "" }
```
Comment (cgrayson) [inline fields/mortgage.json]: No description?

**Comment by cgrayson** [comment]
Otherwise, plus whatever you & Alex are on about, LGTM

File: fields/mortgage.json (hunk 2 above)
Comment (cgrayson) [inline fields/mortgage.json]: I don't know. Range types already have examples, remember. It just seems odd to specify it on one, but not others, unless there's a particular reason that isn't clear here.

Comment (ros3bud) [inline fields/mortgage.json]: I'll take it out.

File: fields/mortgage.json (hunk 3 above)
Comment (ros3bud) [inline fields/mortgage.json]: I'm actually going to remove this one. I don't think it's needed anymore.

File: fields/mortgage.json (hunk 5 above)
Comment (ros3bud) [inline fields/mortgage.json]: Added. That threw me off.. it's in the docs twice. Hopefully when this is merged that will be fixed.

Comment (cgrayson) [inline fields/mortgage.json]: Sorry, what will be hopefully fixed when this is merged?

**Pull request merged**